tracing/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Unreleased
+
+### Changed
+
+- The `instrument` attribute was placed under an on-by-default feature
+  flag.
```

Comment:
Nit: I like to reference the PR numbers in the changelog:
```suggestion
  flag (#603)
```

Reply:
Hehe, yup, I just didn't know it at the time :p